Items/04_00_WeaponAttachments.txt

```diff
@@ -601,6 +601,23 @@ biologic targets and armor.
 by each shot by 20 points, but if the round hits the health pool of a cybernetic
 target, it deals an extra 45 damage.
 
+== Flechette Shells ($2.5 x mag cost) ==
+
+	Flechette (Fla-ch-ette) shells are a shotgun only ammo type, which contain 

```

Comment:
I love the pronunciation guide. I was wondering this myself! :joy:

Reply:
ch is not its own syllable. 
via Merriam-Webster: fle-ˈshet